stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -5606,39 +5606,39 @@ func.func @complex(%arg0: tensor<10x10xf32>, %arg1: tensor<10x10xf32>) -> tensor
 
 // -----
 
-func.func @complex_int_input(%arg0: tensor<10x10xi32>, %arg1: tensor<10x10xi32>) -> tensor<10x10xcomplex<i32>> {
+func.func @complex_i1_i2(%arg0: tensor<10x10xi32>, %arg1: tensor<10x10xi32>) -> tensor<10x10xcomplex<i32>> {
```

Comment:
You may delete all 5 tests here since they are all covered by the ODS following the checklist.